components/ws-proxy/pkg/config/config.go

```diff
@@ -70,7 +70,7 @@ func GetConfig(fn string) (*Config, error) {
 		return nil, xerrors.Errorf("config validation error: %w", err)
 	}
 
-	timeout := time.Minute * 5
+	timeout := time.Second * 45
```

Comment:
pod will mark as crash (?) after around 1 minute not alive, so 5m here is make no sense.

As we plan to remove related code, we could check logs of ws-proxy to verify if feature is disabled